src/pageComponents/search/Results.jsx

```diff
@@ -71,21 +72,24 @@ class Results extends PureComponent {
       } = result;
       const description = truncate(friendlySearchString);
       return (
-        <MediaWrapper key={ i } url={ url }>
-          <Media>
-            <Media.Left align='middle'>
-              <FontAwesome
-                className='resultIcon'
-                name={ faNames[_index] ? faNames[_index] : '' }
-                size='3x'
-              />
-            </Media.Left>
-            <Media.Body>
-              <Media.Heading>{ title }</Media.Heading>
-              <p>{ description }</p>
-            </Media.Body>
-          </Media>
-        </MediaWrapper>
+        <div key={ i }>
```

Comment:
There is no need for this extra `div`, the `key` prop can be applied to `MediaWrapper`

Reply:
Sorry. Didn't notice that.